### PR TITLE
Bugfix/securitypatchfix issue

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: lib/dojos.js [hunk 1]
```diff
@@ -1078,25 +1078,6 @@ exports.register = function (server, options, next) {
     }
   }, {
     method: 'GET',
-    path: options.basePath + '/countries/continents/lat-long',
```
Comment (DanielBrierton) [inline lib/dojos.js]: Purposely removed I assume?

File: lib/dojos.js [hunk 2]
```diff
@@ -1134,25 +1115,6 @@ exports.register = function (server, options, next) {
       }
     }
   }, {
-    method: 'GET',
```
Comment (DanielBrierton) [inline lib/dojos.js]: And again, purposely removed?

File: lib/dojos.js (hunk 1 above)
Comment (Wardormeur) [inline lib/dojos.js]: unused y

File: lib/dojos.js (hunk 2 above)
Comment (Wardormeur) [inline lib/dojos.js]: unused y

**Pull request merged**